src/quest_manager/models.py

```diff
@@ -975,3 +988,55 @@ def get_minutes_to_complete(self):
 
         minutes = (self.first_time_completed - self.timestamp).total_seconds() / 60
         return minutes
+
+
+class QuestionType(models.Model):
+    id = models.CharField(
+        max_length=100,
+        primary_key=True,
+        help_text="Short, memorable, unique identifier for a question type e.g. 'short_answer', 'file_upload', 'multiple_choice'.",
+        serialize=False
+    )
+    name = models.CharField(
+        max_length=255,
+        help_text="The name of the question type e.g. 'Short Answer', 'File Upload', 'Multiple Choice'."
+    )
+    description = models.TextField(blank=True, null=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Question(models.Model):
+    instructions = models.TextField()
+    solution_text = models.TextField(null=True, blank=True)
+    solution_file = RestrictedFileField(blank=True, null=True, upload_to='quest_questions/')
+    marker_notes = models.TextField(null=True, blank=True)
+
+    required = models.BooleanField(default=True)
+    datetime_created = models.DateTimeField(auto_now_add=True)
+    datetime_last_edit = models.DateTimeField(auto_now=True)
+    ordinal = models.PositiveIntegerField(default=1)
+
+    quest = models.ForeignKey(Quest, on_delete=models.CASCADE)
+    type = models.ForeignKey(QuestionType, on_delete=models.CASCADE, null=True, blank=True)
+
+    class Meta:
+        ordering = ['ordinal']
```

Comment:
Make sure this sorts the correct direction (lower number first) and ensure there is a test for that.